tardis/montecarlo/montecarlo_numba/base.py

```diff
@@ -134,6 +135,31 @@ def montecarlo_radial1d(
     # Condition for Checking if RPacket Tracking is enabled
     if montecarlo_configuration.RPACKET_TRACKING:
         runner.rpacket_tracker = rpacket_trackers
+        runner.rpacket_tracker_df = obj_list_to_dataframe(rpacket_trackers)
+
+
+def obj_list_to_dataframe(trackers_list):
+    rtracker_dict_list = []
+    index_array = []
+    for i in trackers_list:
```

Comment:
Avoid `i`, `j`, use descriptive name to specify what you are iterating over. I think Jack also suggested it earlier.